Modix/Modules/LmgtfyModule.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Net;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Commands;
+
+namespace Modix.Modules
+{
+    [Name("Let Me Google That For You")]
+    [Summary("Sometimes we all need that little extra push")]
+    public sealed class LmgtfyModule : ModuleBase
```

Comment:
Is there a particular reason this is `sealed`? This is definitely a personal preference, but I try not to place unnecessary restrictions on how code can be used.